File: python/p1.py
i = 3
ans = 0
while i < 1000: 
    if i % 3 == 0:
        if i % 5 == 0:
            ans += i
        else:
            ans += i
    elif i % 5 == 0:
            ans += i 
    i += 1
print("ans = %d", ans)

File: C++/p1.cpp
#include <iostream>

using namespace std;

int main(void)
{
    int ans = 0;
    for (int i = 3; i < 1000; i++) ans += i % 3 == 0 || i % 5 == 0 ? i : 0;
    cout << "ans = " << ans << endl;
    return 0;
}
